File: multiplayer/angular-frontend/src/app/game-lobby/model/GameRoom.ts
export class GameRoom {
  public roomId: string;
  public gameType: string;
  createdBy: string;
  playersJoined: string;

  constructor(roomId: string, gameType: string, createdBy: string, playersJoined: string) {
    this.roomId = roomId;
    this.gameType = gameType;
    this.createdBy = createdBy;
    this.playersJoined = playersJoined;
  }
}

File: multiplayer/spring-boot-backend/src/main/java/com/murlan/game/shared/model/GameRoom.java
package com.murlan.game.shared.model;

public class GameRoom {
    private String roomId;
    private String gameType;
    private String createdBy;
    private String playersJoined;

    public String getRoomId() {
        return roomId;
    }

    public void setRoomId(String roomId) {
        this.roomId = roomId;
    }

    public String getGameType() {
        return gameType;
    }

    public void setGameType(String gameType) {
        this.gameType = gameType;
    }

    public String getCreatedBy() {
        return createdBy;
    }

    public void setCreatedBy(String createdBy) {
        this.createdBy = createdBy;
    }

    public String getPlayersJoined() {
        return playersJoined;
    }

    public void setPlayersJoined(String playersJoined) {
        this.playersJoined = playersJoined;
    }
}
